cli/portforward.go

```diff
@@ -137,7 +137,7 @@ func (r *RootCmd) portForward() *serpent.Command {
 				listeners[i] = l
 			}
 
-			stopUpdating := client.UpdateWorkspaceUsageContext(ctx, workspace.ID)
+			stopUpdating := client.UpdateWorkspaceUsageContext(ctx, workspace.ID, codersdk.PostWorkspaceUsageRequest{})
```

Comment:
non-blocking: We might want to expose this as an app named "port-forward" or similar?

Reply:
Let's leave it for now. I'd like to avoid any insights changes in the course of this refactor.